charts/tekton-pipelines/README.md

```diff
@@ -25,19 +26,20 @@ __|_____________
 
 ## `chart.deprecationWarning`
 
+
 ## `chart.name`
 
 saritasa-tekton-pipelines
 
 ## `chart.version`
 
-![Version: 0.1.15](https://img.shields.io/badge/Version-0.1.15-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v0.28.2](https://img.shields.io/badge/AppVersion-v0.28.2-informational?style=flat-square)
+![Version: 0.1.16](https://img.shields.io/badge/Version-0.1.16-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v0.28.2](https://img.shields.io/badge/AppVersion-v0.28.2-informational?style=flat-square) 
 
 ## Maintainers
 
 | Name | Email | Url |
 | ---- | ------ | --- |
-| Saritasa |  | https://www.saritasa.com/ |
+| Saritasa |  | <https://www.saritasa.com/> |
 
 ## `chart.description`
```

Comment:
You should add the description and example about how to use the new property.

Reply:
added